src/components/button/button.scss

```diff
@@ -0,0 +1,108 @@
+/** Imports **/
+
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+/** Variables **/
+
+$color-secondary-active: rgba(0, 0, 0, 0.12);
+$timing-primary: 0.2s;
+$secondary-underline-height: 2px;
+$secondary-underline-offset: 8px;
+$secondary-underline-offset-hover: 4px;
+
+.kui-theme--light {
+  --color-button--active: #{rgba($color-dark, 0.85)};
+  --color-button--outline: #{$color-dark};
+  --color-button__text--hover: #{rgba($color-light, 0.85)};
+  --color-button__text: #{rgba($color-dark, 0.85)};
+}
+
+.kui-theme--dark {
+  --color-button--active: #{rgba($color-light, 0.85)};
+  --color-button--outline: #{$color-light};
+  --color-button__text--hover: #{rgba($color-dark, 0.85)};
+  --color-button__text: #{rgba($color-light, 0.85)};
+}
+
+.kui-button {
```

Comment:
Please remove references of `kui` and rename all css classnames for this component to either `kedro-button` or just `button`.